.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,26 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+	"image": "mcr.microsoft.com/devcontainers/python:3",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				// Settings for the python extension.
+				"python.defaultInterpreterPath": "${workspaceFolder}/.venv",
+				"python.terminal.activateEnvInCurrentTerminal": true
+			}
+		},
+		"codespaces": {
+			"openFiles": [
+				".devcontainer/welcome-message.txt"
+			]
```

Comment:
@davidism Rather than introduce complexity by trying to fix the issue with the welcome text being cleared from the terminal, I think a simpler solution is to open the file at launch.

If that works for you, we can move the `welcome-message.txt` to a better home in the project, or replace it with a codespaces-specific contributing guide. We can also just open `CONTRIBUTING.rst` at launch, and contributors can read through to the relevant section.

Reply:
Any thoughts on this @davidism?

Reply:
As a maintainer that gets a ton of notifications and has a lot to deal with besides this project, I really prefer not to be pinged only to check for updates if nothing has changed. You'll know my thoughts when I interact with the PR :wink:

Reply:
Turns out there's a simple fix here with a vscode setting. Should be solved now!